src/steampunk_scanner/commands/scan.py

```diff
@@ -56,20 +63,21 @@ def _parser_callback(args: argparse.Namespace):
 
     response = client.post("/scan-tasks", input_tasks)
     if response.ok:
+        workspace.discover()
```

Comment:
Here we only list workspace properties. They are not inclueded in API call.

Reply:
Yes, true we only call the method to get user's workspace info and we do nothing with it and it can stay like that until the API will be able to accept that info do something with it.

If you think that calling this method has no sense I can remove it until we actually will need it.